bucket/teamspeak3.json

```diff
@@ -24,7 +24,14 @@
             ]
         }
     },
-    "post_install": "Remove-Item \"$dir\\`$PLUGINSDIR\" -Force -Recurse",
+    "pre_install": [
+        "Remove-Item -Path \"$dir\\.*\" -Force -Recurse",
+        "Remove-Item -Path \"$dir\\CERTIFICATE\" -Force -Recurse",
+        "7z x -y \"$dir\\[0]\" -o\"$dir\" \"-xr!`$PLUGINSDIR\" | Out-Null",
```

Comment:
FYI, 7z is needed to install this package anyways as it is needed for the first extraction out of two, as such the command is readily available at this point.